### ISSUE TITLE
FR: Add Class to Style Active Note in BC Matrix

### ISSUE THREAD

**Issue by bodiequirk** [SkepticMystic/breadcrumbs#245]
Breadcrumbs has become how I navigate my Obsidian vault, and I love it! The only thing I still need is the ability to see which note I am on based on its styling in matrix view. Like websites, it's sometimes helpful to glance at the menu to remember which page you are on (related to is siblings) and have it stand out. I would love to be able to do that with Breadcrumbs.

Thanks!

**Comment by SkepticMystic** [comment]
I've added this in the latest release :)
The entire list item has the class `BC-active-note`

**Comment by bodiequirk** [comment]
Thank you! I am not able to see it work, however (this could be due to my weak CSS skills).

When I use the `.internal-link.BC-Link` class I am able to target the Matrix CSS properties and change them. However, I am not able yet to target just the active note with the `.BC-active-note` class.

I have tried every variation to no avail. I have tried:

`.BC-active-note.internal-link.BC-Link`
`.BC-active-note .internal-link.BC-Link`
`.internal-link.BC-Link .BC-active-note`
`.BC-active-note` alone

Is there something else I should try? Thanks for your help!
